packages/dataviews/src/filter-summary.js

```diff
@@ -73,6 +73,10 @@ function WithSeparators( { children } ) {
 }
 
 export default function FilterSummary( { filter, view, onChangeView } ) {
+	if ( view.type === LAYOUT_LIST ) {
```

Comment:
Similar with the above comment. it feels to me we should not make such explicit checks. Maybe this are kind of a config of a layout(showFilterSummary, etc..)? 🤔

Reply:
When you suggest making it a prop, are you thinking of lifting it up all the way to the `DataViews` component for consumers to configure?

Reply:
Not sure if it's what you were looking for, but since you mentioned it, I revisited this code and prepared a follow-up to lift the visibility control to the parent https://github.com/WordPress/gutenberg/pull/57056